src/main/java/practice/CandidateValidator.java

```diff
@@ -1,5 +1,39 @@
 package practice;
 
-public class CandidateValidator {
-    //write your code here
+import java.util.function.Predicate;
+import model.Candidate;
+
+public class CandidateValidator implements Predicate<Candidate> {
+    private static final String UKRAINIAN_NATIONALITY = "Ukrainian";
+    private static final String SEPARATOR = "-";
+    private static final int AGE_FOR_VOTING = 35;
+    private static final int LIVE_IN_UKR = 10;
+
+    @Override
+    public boolean test(Candidate candidate) {
+        if (candidate.getAge() >= AGE_FOR_VOTING
+                && candidate.isAllowedToVote()
+                && candidate.getNationality().equals(UKRAINIAN_NATIONALITY)
+                && livingInUkr(candidate) >= LIVE_IN_UKR) {
+            return true;
+        }
+        return false;
+    }
```

Comment:
Why do you need `if` block? 
```suggestion
        return candidate.getAge() >= AGE_FOR_VOTING
                && candidate.isAllowedToVote()
                && candidate.getNationality().equals(UKRAINIAN_NATIONALITY)
                && livingInUkr(candidate) >= LIVE_IN_UKR;
```